rkt/images.go

```diff
@@ -224,49 +222,101 @@ func (f *fetcher) fetchSingleImage(img string, asc string, discover bool) (strin
 		return "", fmt.Errorf("unable to access %q: %v", img, err)
 	}
 
-	if discover && u.Scheme == "" {
+	switch u.Scheme {
+	// image name
+	case "":
+		return f.fetchImageByName(img, ascFile)
+	case "http", "https", "file", "docker":
+		return f.fetchImageByURL(u, ascFile)
+	}
+
+	return "", fmt.Errorf("rkt only supports image name and http, https, docker or file URLs (%s)", img)
```

Comment:
`image name` sounds obscure to me, as I think it can also be a hash (or image id). But I don't have a good answer for this as well, `image name, hash` ?